homework-submission/todoapp/src/Components/TodoElement.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+class TodoElement extends React.Component {
+  state = {
+    isEdit: false
+
+  }
+  handleEditButton = () => {
+    this.setState({
+      isEdit: true
+    })
+  }
+  handleCancelButton = () => {
+    this.setState({
+      isEdit: false
+    })
+  }
+  handleUpdate = (e) => {
+    e.preventDefault()
+    let oldTask = this.props.description
+    let newTask = this.editField.value
+    let newDate = this.secondEditField.value
+    this.props.updateTodo(oldTask, newTask, newDate)
+    this.setState({
+      isEdit: false
+
+    })
+
+  }
+
+  render() {
+    let TodoView
+    if (!this.state.isEdit) {
+      TodoView = (
+        <div className="todo" key={this.props.unique}>
+          <input type='checkbox' id='description' defaultChecked={this.props.done} onChange={() => this.props.handleClick(this.props.unique)} />
+          <span className={this.props.done ? 'checked' : 'notChecked'}>{this.props.description}, {this.props.deadline}</span>
+          <button onClick={this.handleEditButton}>Edit</button>
+          <button onClick={() => this.props.deleteTodo(this.props.unique)}>Delete</button>
+        </div >
+      )
+    } else {
+      TodoView = (
+        <form onSubmit>
+          <div classNAme="todo editMode" key={this.props.unique}>
```

Comment:
if you checked your browser in the inspector tab you will find `classname` all small letters, it's because of `classNAme` 😬